File: server/models/status_const.go
package models

const (
	// 用户在线状态常量
	QMe         int32 = 1
	Online      int32 = 2
	Away        int32 = 3
	Busy        int32 = 4
	DontDisturb int32 = 5
	OffLine     int32 = 6

	// 请求状态常量
	Failure       int32 = 0
	Successful    int32 = 1
	UserExists    int32 = 2
	RepeatLogin   int32 = 3
	UserNotExists int32 = 4
	WrongPassword int32 = 5
)

File: client/GoGoTalk/Models/StatusConst.cs
using System;

namespace GoGoTalk.Models
{
    public class StatusConst
    {
        public enum ResultCode : Int32
        {
            /// <summary>
            /// 失败
            /// </summary>
            Failure = 0,

            /// <summary>
            /// 成功
            /// </summary>
            Successful = 1,

            /// <summary>
            /// 用户已存在
            /// </summary>
            UserExists = 2,

            /// <summary>
            /// 重复登录
            /// </summary>
            RepeatLogin = 3,

            /// <summary>
            /// 用户不存在
            /// </summary>
            UserNotExists = 4,

            /// <summary>
            /// 密码错误
            /// </summary>
            WrongPassword = 5,
        };
    }
}
